EIPS/eip-3333.md

```diff
@@ -0,0 +1,51 @@
+---
+eip: 3333
+title: Increase block rewards to 3 ETH, with 2 Year Decay to 1 ETH Scheduled
+author: Michael D. Carter (BitsBeTrippin)
+discussions-to: https://discord.gg/3sjBbWKM4E (ETH R&D 1559-mining)
```

Comment:
Discussions-to links need to be to a semi-permanent asynchronous discussion forum.  A thread at https://ethereum-magicians.org/ is strongly preferred, though a new issue in this GitHub repository is considered an acceptable alternative if you unable or unwilling to create a thread over there.

Reply:
Will add